<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>821</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 250, 254);
font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="dashBoard_widget" native="true">
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	
background-color: rgb(0, 173, 181);
}
QPushButton{
	text-align:left;
	height: 70px;
	border: none;
	font: 700 14pt&quot;Sans Serif Collection&quot;;
	
	color: rgb(255, 255, 255);
	padding-left: 10px;
	border-top-left-radius:10px;
	border-bottom-left-radius:10px;
}
QPushButton:checked{
	background-color: rgb(255, 255, 255);
	color:#1F95EF;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QPushButton" name="admin_PushButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Admin</string>
           </property>
           <property name="icon">
            <iconset resource="../../assets/resources.qrc">
             <normaloff>:/icons/icons/admin.png</normaloff>:/icons/icons/admin.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">dashBoard_Buttons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="medicine_pushButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Medicine</string>
           </property>
           <property name="icon">
            <iconset resource="../../assets/resources.qrc">
             <normaloff>:/icons/icons/medicine.png</normaloff>:/icons/icons/medicine.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">dashBoard_Buttons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="patient_pushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Patient</string>
           </property>
           <property name="icon">
            <iconset resource="../../assets/resources.qrc">
             <normaloff>:/icons/icons/patient.png</normaloff>:/icons/icons/patient.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">dashBoard_Buttons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="log_pushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Logs</string>
           </property>
           <property name="icon">
            <iconset resource="../../assets/resources.qrc">
             <normaloff>:/icons/icons/log.png</normaloff>:/icons/icons/log.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">dashBoard_Buttons</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="title_Label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font:700 16pt &quot;Microsoft YaHei UI&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Welcome to ADC</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="main_stackwidget">
        <property name="styleSheet">
         <string notr="true">Qlabel{
	font: 700 16pt &quot;Segoe UI&quot;;
	color: rgb(0, 85, 255);
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QGridLayout" name="gridLayout_2"/>
        </widget>
        <widget class="QWidget" name="page_2"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../assets/resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="dashBoard_Buttons"/>
 </buttongroups>
</ui>
